books/projects/x86isa/machine/instructions/pcmp.lisp

```diff
@@ -147,3 +147,108 @@
        ;; Update the instruction pointer.
        (x86 (write-*ip proc-mode temp-rip x86)))
       x86))
+
+(define pcmpgt ((result-width natp)
+                (el-width posp)
+                (a natp)
+                (b natp))
+  :prepwork ((local (include-book "arithmetic-5/top" :dir :system)))
+  :guard (equal (mod result-width el-width) 0)
+  :returns (result (unsigned-byte-p result-width result)
+                   :hyp (and (natp result-width)
+                             (<= (pos-fix el-width) result-width)
+                             (equal (mod result-width (pos-fix el-width)) 0))
+                   :hints (("Goal" :in-theory (disable unsigned-byte-p
+                                                       bitops::logapp-of-i-0
+                                                       acl2::prefer-positive-addends-<))))
+  :measure (nfix result-width)
+  (b* ((result-width (nfix result-width))
+       (el-width (pos-fix el-width))
+       (a (nfix a))
+       (b (nfix b))
+       ((when (zp result-width)) 0)
+       (el-equal? (> (logext el-width a)
```

Comment:
`el-greater?` may be a better name for this variable.

Reply:
Whoops, copy pasta error